esmvaltool/utils/recipe_test_workflow/doc/source/assessment_areas.rst

```diff
@@ -0,0 +1,11 @@
+.. (C) Crown Copyright 2022, the Met Office.
+
+**************************
+Available assessment areas
+**************************
```

Comment:
```suggestion
************************
Currently tested recipes
************************
```

Reply:
Also, please rename this file to `tested_recipes.rst` :)

Reply:
5f7ae32f5 done